### DISCUSSION TITLE
How to update Flowise version in Railway?

### DISCUSSION THREAD

**Discussion by home-apps** [FlowiseAI/Flowise#1284] [Q&A] [answered]
I have a newbie question. I deployed Flowise to Railway a couple of months ago. I see that Flowise has been updates with many really good features, however, my Flowise dashboard does not show any of those features. How do I update my existing Railway installation? Can you please help me to understand the steps to take so that I can update it now and for future versions? Thank you

**Comment by yavisht** [comment, marked as answer]
@home-apps 

### Step 1: Backup your database 
<img width="301" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/6112201/661761c8-ee01-4076-8039-6935399ec10e">

### Step2: Click on your railway project
<img width="351" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/6112201/393e5e34-a285-4cfe-9bec-6234c8ed2aec">

### Step 3: Redeploy your latest deployment
<img width="349" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/6112201/a85800c6-b60b-4223-8b9c-dfd6d897180d">

### IMPORTANT: 
Make sure you have all the env variables set so that the data is not wiped. Adding/Changing env variables will redeploy the code.

See here:
https://github.com/FlowiseAI/Flowise/discussions/1037#discussioncomment-7258470
